sdk/agricultureplatform/arm-agricultureplatform/tsconfig.browser.config.json

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["./tsconfig.test.json", "../../../tsconfig.browser.base.json"]
+  "extends": "../../../tsconfig.browser.base.json",
+  "compilerOptions": {
+    "paths": {
+      "@azure/arm-agricultureplatform": ["./dist/browser/index.d.ts"],
+      "@azure/arm-agricultureplatform/*": ["./dist/browser/*"],
```

Comment:
Would this allow a browser test to import internal things from, e.g., `@azure/arm-agricultureplatform/src/internal`?